packages/core/src/App/Components/Routes/route-with-sub-routes.jsx

```diff
@@ -28,7 +28,10 @@ const RouteWithSubRoutes = route => {
             }
             result = <Redirect to={to} />;
         } else if (is_valid_route && route.is_authenticated && !route.is_logged_in && !route.is_logging_in) {
-            redirectToLogin(route.is_logged_in, getLanguage());
+            const url_search_params = new URLSearchParams(window.location.search);
+            if (url_search_params.has('is_redirecting')) {
```

Comment:
```suggestion
            if (!url_search_params.has('is_redirecting')) {
```
I think we should check if the `url_search_params` doesn't contain `is_redirecting` flag

Reply:
I changed the approach to save the flag to localstorage and removing it once a delayed redirect is done. 🙏🏼